ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/commands/CommandCompletionServiceIT.scala

```diff
@@ -79,7 +87,7 @@ class CommandCompletionServiceIT
             ctx.ledgerId,
             applicationId,
             Seq(party),
-            LedgerOffset(Boundary(LEDGER_BEGIN)))
+            Some(LedgerOffset(Boundary(LEDGER_BEGIN))))
```

Comment:
mind that this is coming to avoid boiler-plating: https://github.com/digital-asset/daml/pull/1863/files#diff-1f9584cad6e35fdc752fe77e66266729R32

Reply:
I'll set up a reminder to fix this as soon as #1863 is merged, thanks.